src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.ecommercee.domain;
+
+
+import lombok.*;
```

Comment:
ja nigdy nie stosuje importu z * i uważam, że to jest zła praktyka. Można przestawić IntelliJ aby tego też nie robił.